### PR TITLE
Feat/signature

### PR DESCRIPTION
**Pull request by taylorjdawson**

### CODE REVIEW

File: test/signature.test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import test from "ava"
-import {ERROR_MESSAGE_SIGNATURE_NO_HASH as NO_HASH, ADDRESS} from "../src/constants";
+import {ERROR_MESSAGE_SIGNATURE_NO_HASH as NO_HASH} from "../src/constants";
 import {setUpPolly, getNewWeb3DataInstance} from "./utils";
 
-/**********************************
+/**********************************gs
```
Comment (TrevorJTClarke) [inline test/signature.test.js]: @taylorjdawson typo?

File: src/contract.js [hunk 2]
```diff
@@ -33,24 +33,36 @@ class Contract {
...
 
-  getAudit(hash, filterOptions) {
+  /**
+   * Retrieves the vulnerabilities audit for the specified contract (if available).
+   *
+   * @param hash - The contract address.
+   * @returns The vulnerabilities audit for the specified contract.
+   * @example const audit = await web3data.contract.getAudit('0x06012c8cf97bead5deae237070f9587f8e7a266d')
+   */
+  getSecurityAudit(hash) {
```
[the remark this answers is not in the document]
Comment (taylorjdawson) [reply src/contract.js]: I can just keep both:
```js
getAudit(hash) {
  return this.getSecurityAudit(hash)
}
```

File: test/signature.test.js (hunk 1 above)
Comment (taylorjdawson) [reply test/signature.test.js]: fixed

File: src/contract.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (TrevorJTClarke) [reply src/contract.js]: sounds good

[the remark this answers is not in the document]
Comment (TrevorJTClarke) [reply src/contract.js]: @taylorjdawson still needs the alias

**Pull request merged**